<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Select_Offset</class>
 <widget class="QDialog" name="Select_Offset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>349</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Choose an Offset Duration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/vis-1-32.png</normaloff>:/icons/icons/vis-1-32.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>music21 Duration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="music21_durat">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QToolButton" name="btn_0_25">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/sixteenth.png</normaloff>:/icons/icons/sixteenth.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="btn_2">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/half.png</normaloff>:/icons/icons/half.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="btn_1">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/quarter.png</normaloff>:/icons/icons/quarter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="btn_4">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/whole.png</normaloff>:/icons/icons/whole.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="btn_8">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/breve.png</normaloff>:/icons/icons/breve.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="btn_0_5">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/eighth.png</normaloff>:/icons/icons/eighth.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="btn_0_125">
        <property name="text">
         <string>32nd</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/thirty_second.png</normaloff>:/icons/icons/thirty_second.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="btn_0_0625">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/sixty_fourth.png</normaloff>:/icons/icons/sixty_fourth.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="btn_none">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>all</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>332</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_submit">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/show_checkmark.png</normaloff>:/icons/icons/show_checkmark.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_submit</sender>
   <signal>clicked()</signal>
   <receiver>Select_Offset</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
